src/Sylius/Bundle/FlowBundle/Process/Builder/ProcessBuilder.php

```diff
@@ -79,6 +79,10 @@ public function add($name, $step)
             throw new \InvalidArgumentException('Step added via builder must implement "Sylius\Bundle\FlowBundle\Process\Step\StepInterface"');
         }
 
+        if (false === $step->isActive()) {
```

Comment:
ok, in Symfony CMF we do `=== false` generally. It makes more sense to me as it means that we implicitly do not support the subject returning non-boolean falsy values. But I really don't feel strongly about it - and often use the negator myself, its just a matter of setting a precedent for future contributions :) Will change.